oauth2client/contrib/django_util/views.py

```diff
@@ -170,21 +170,22 @@ def oauth2_authorize(request):
          A redirect to Google OAuth2 Authorization.
     """
     return_url = request.GET.get('return_url', None)
+    if not return_url:
```

Comment:
Prefer `is None`/`is not None` when comparing to the `None` object.

If `request.GET.get('return_url', None)` is the empty string, that's a bug, right? And by unintentionally sweeping it along the same code path as `request.GET`-does-not-contain-a-key-value-mapping-for-key-`return_url` we make finding and fixing the bug harder, right?